RecoParticleFlow/PFTracking/src/PFTrackAlgoTools.cc

```diff
@@ -237,9 +237,10 @@ namespace PFTrackAlgoTools {
                         const std::vector<double>& DPtovPtCut,
                         const std::vector<unsigned>& NHitCut,
                         bool useIterTracking,
+                        reco::TrackBase::TrackQuality trackQuality,
                         bool debug) {
```

Comment:
I notice that the ```debug``` option is not needed any more by this method. It was introduced by commit https://github.com/cms-sw/cmssw/commit/5eafe51907989c8245f37f5fe46a7d8ce9a0c135#diff-05a1b01a408107e62fa1938501541abd  , but then made immediately after useless when moving all ```cout``` comment lines to ```LogDebug``` with https://github.com/cms-sw/cmssw/commit/4c64ac90711ec6a8eb9ac9e8c4a763035219887d#diff-05a1b01a408107e62fa1938501541abd

You can profit of this PR to remove that method parameter

Reply:
@perotta done in commit below